scripts/release/asset_publish.py

```diff
@@ -11,26 +11,25 @@
 import sys
 from pathlib import Path
 from string import Template
-from subprocess import PIPE, STDOUT, run
+from subprocess import PIPE, run
```

Comment:
If you accept my suggestion about `capture_output=True` below:
```suggestion
from subprocess import run
```